src/components/RecentSubmission.js

```diff
@@ -3,13 +3,17 @@ import PropTypes from 'prop-types';
 import './RecentSubmission.css';
 
 const RecentSubmission = (props) => {
```

Comment:
Just to note the tests for this component are passing, so you didn't need to skip them.